src/pineko/fonll.py

```diff
@@ -90,9 +90,8 @@ def produce_combined_fk(
     fonll_info = FONLLInfo(ffns3, ffn03, ffns4til, ffns4bar, ffn04, ffns5til, ffns5bar)
 
     theorycard_constituent_fks = fonll_info.theorycard_no_fns_pto
-    if theorycard_constituent_fks["DAMP"] == 0:
+    if theorycard_constituent_fks["DAMP"] == "False":
```

Comment:
This is weird... why a string?

Reply:
You are right, fixed here [3ca875c](https://github.com/NNPDF/pineko/pull/99/commits/3ca875cd48f5b0e72befed2991924f1d45ac0b78)